api/v1beta1/common.go

```diff
@@ -93,10 +88,10 @@ type NodeTemplate struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:io.kubernetes:Secret"}
 	AnsibleSSHPrivateKeySecret string `json:"ansibleSSHPrivateKeySecret"`
 
-	// NetworkConfig - Network configuration details. Contains os-net-config
-	// related properties.
+	// NetworkAttachments is a list of NetworkAttachment resource names to pass to the ansibleee resource
```

Comment:
Why're we adding this back?

Reply:
Must have been added back in the rebase. Removing this and the `DeployStrategySection` you mentioned below.